File: (8 kyu) Reversed Strings.rs
/// Reversed Strings (8 kyu)
/// https://www.codewars.com/kata/reversed-strings
/// Complete the solution so that it reverses the string value passed
/// into it.
/// solution("world") // returns "dlrow"
fn solution(phrase: &str) -> String {
    // #1
    // phrase.chars().rev().collect::<String>()
    // #2
    phrase.chars().rev().collect()
}

File: (8 kyu) Reversed Strings/(8 kyu) Reversed Strings.js
/** Reversed Strings (8 kyu)
 * https://www.codewars.com/kata/reversed-strings
 *
 * Complete the solution so that it reverses the string value passed
 * into it.
 * solution('world'); // returns 'dlrow'
 */
// #1
// function solution(str){
//     return str.split("").reverse().join("");
// }
// #2
// const solution = (s) => s.split``.reverse().join``
// #3
const solution = (s) => [...s].reverse().join``;
